docs/topics/windows-provisioning-scripts.md

```diff
@@ -0,0 +1,29 @@
+# Windows Provisioning Scripts
+
+## Overview
+
+Due to requirements around signing executable code on Windows nodes some powershell scripts are are delivered via a .zip file during node provisioning.
+This document details how to update and test changes to these files.
+
+## Testing
+
+Changes to the provisioning scripts packaged in the .zip can be tested by:
+
+- Running [scripts/build-windows-provisioning-scripts.sh](../../scripts/build-windows-provisioning-scripts.sh) which will create a .zip and upload it to a storage account
+- Create/update a cluster definition file and set `windowsProfile.provisioningScriptsPackageURL' to point to the uploaded back.
+- Perform a cluster deployment with aks-engine.
+
+## Updates
+
+- Check in script updates to [staging/provisioning/windows](../../staging/provisioning/windows)
+- Create a new **versioned** zip with the contents of [staging/provisioning/windows](../../staging/provisioning/windows) and upload it to a storage account
```

Comment:
Why versioned?

Reply:
We need to have different files names for the caching logic to pick the correct files from the VHD.

Reply:
Can you include that explanation in the doc?

Reply:
Added.